backend/app/core/config.py

```diff
@@ -13,10 +13,20 @@ class ServerSessionsSettings(BaseSettings):
     expires_in: int
 
 
+class MatrixBotSettings(BaseSettings):
+    homeserver: str
+    user_id: str
+    password: str
+    device_id: str
```

Comment:
If we only support access token we can replace all three of these settings with just an access token. Making it easier for us and easier for users to configure.